portfolio/src/main/webapp/style.css

```diff
@@ -1,17 +1,31 @@
+body{
```

Comment:
For an extra cherry on top, alphabetize properties (as suggested in these [CSS formatting best practices](https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules)). [Related commentary here.](https://github.com/stephypy/step/pull/2#pullrequestreview-420152060)